samples/bluetooth/direction_finding_connectionless_rx/README.rst

```diff
@@ -13,6 +13,7 @@ periodic advertising PDUs.
 Requirements
 ************
 
+* nRF52833DK board with nRF52833 SOC
 * nRF52833DK board with nRF52833 SOC
```

Comment:
Duplicated line

Reply:
I guess one of them should be with nRF52820 SOC?

Reply:
Yes, it should be nRF52820, thank you. It is corrected.